config/locales/en.yml

```diff
@@ -398,14 +410,6 @@ en:
       Don't send out the response until <strong>after %{escalation_deadline}</strong>.
     response_upload_error: Errors detected with uploaded files.
     response_upload_blank?: Please select the file(s) you used in your response.
-    respond_point_of_no_return_html: |
-      <p class="notice">
-        <i class="icon icon-important">
-          <span class="visually-hidden">Important</span>
-        </i>
-        <strong>You can't update a response after marking it as sent.</strong>
-      </p>
-
```

Comment:
Why was this removed?

Reply:
it wasn't in the designs and we decided to make the FOI and ICO page the same